pytorch/training/docker/1.9/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,149 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON=python3
+# Add arguments to achieve the version, python and url
+ARG PYTHON_VERSION=3.8.10
+ARG OPEN_MPI_VERSION=4.0.1
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.9
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+# Force stdin, stdout and stderr to be totally unbuffered. Good for logging
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/conda/lib"
+ENV PATH=/opt/conda/bin:$PATH
+ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
+ENV DGLBACKEND=pytorch
+
+ARG PT_TRAINING_URL=https://aws-pytorch-cicd-v3-binaries.s3-us-west-2.amazonaws.com/r1.9.0_aws_v3/20210606-211844/45295974afdd56106c95ffae92ff0ebdc5efc6ab/cpu/torch-1.9.0-cp38-cp38-manylinux1_x86_64.whl
+ARG PT_TORCHVISION_URL=https://torchvision-build-rc.s3-us-west-2.amazonaws.com/1.9.0/cpu/torchvision-0.9.1-cp38-cp38-linux_x86_64.whl
```

Comment:
Any reason why we don't add TorchAudio to this list? Seems like PyTorch officially asks customers to install both [`audio` and `vision` when installing `torch`](https://pytorch.org/get-started/locally/)

Reply:
We did not include it before and did not receive any customer request to include it until now. We can consider adding it next release if there is customer request.